### PR TITLE
Delete CODE_CONVENTIONS.md

### LINKED ISSUE #6: Продумать Workflow issues/projects

### PR DESCRIPTION
**Pull request by File5**
closes #6

### CODE REVIEW

File: CODE_CONVENTIONS.md [hunk 1]
```diff
@@ -1,7 +0,0 @@
-# Конвенции кодирования
-
```
Comment (jsjson0) [inline CODE_CONVENTIONS.md:2]: Single comment

File: CODE_CONVENTIONS.md [hunk 2]
```diff
@@ -1,7 +0,0 @@
...
-## Часть 1
-## Часть 2
-
```
Comment (jsjson0) [inline CODE_CONVENTIONS.md:5]: Start a review

**Review by jsjson0 — CHANGES_REQUESTED** [review]

File: CODE_CONVENTIONS.md (hunk 1 above)
Comment (File5) [reply CODE_CONVENTIONS.md:2]: Yea

File: CODE_CONVENTIONS.md (hunk 2 above)
Comment (File5) [reply CODE_CONVENTIONS.md:5]: Review...

Comment (File5) [reply CODE_CONVENTIONS.md:5]: It's ok

**Review by jsjson0 — APPROVED** [review]

**Pull request merged**